index.html

```diff
@@ -0,0 +1,531 @@
+        </div>
+        <ul class="header__mobile-nav">
+          <li class="header__mobile-item header__mobile-item--second">Home</li>
+          <li class="header__mobile-item header__mobile-item--second">
+            Product
+          </li>
+          <li class="header__mobile-item header__mobile-item--second">
+            Pricing
+          </li>
+          <li class="header__mobile-item header__mobile-item--second">
+            Contact
+          </li>
+        </ul>
+      </div>
+    </header>
+    <main class="main">
+      <section class="banner">
+        <div class="banner__wrapper">
+          <h1 class="heading heading--xl heading--center">
+            Experts are here solve your business problem.
+          </h1>
+          <p class="text text--center text--max-lg text--secondary">
+            We know how large objects will act, but things on a small scale just
+            do not act that way.
+          </p>
+          <div class="banner__actions">
+            <button class="btn">
+              <span class="btn__text">Get Quote Now</span>
+            </button>
+            <button class="btn btn--secondary">
+              <span class="btn__text btn__text--secondary">Learn More</span>
+            </button>
+          </div>
+        </div>
+      </section>
+      <section class="service">
+        <header class="service__header">
+          <h2 class="heading heading--md heading--center">WHY CHOOSE US</h2>
+          <p class="text text--center text--max text--secondary">
+            Problems trying to resolve the conflict between the two major realms
+            of Classical physics: Newtonian mechanics
+          </p>
+        </header>
+        <ul class="list">
+          <li class="list__item">
+            <div class="service__card">
+              <div class="service__card-header">
+                <img src="./assets/toolbox.svg" alt="" />
+                <h5 class="heading__sm">Digital Marketing</h5>
+              </div>
+              <p class="text text--secondary">
+                the quick fox jumps over the lazy dog the quick fox jumps over
+                the lazy dog
+              </p>
+            </div>
+          </li>
+          <li class="list__item">
+            <div class="service__card">
+              <div class="service__card-header">
+                <img src="./assets/shop.svg" class="service__card-img" alt="" />
+                <h5 class="heading__sm">Digital Marketing</h5>
+              </div>
+              <p class="text text--secondary">
+                the quick fox jumps over the lazy dog the quick fox jumps over
+                the lazy dog
+              </p>
+            </div>
+          </li>
+          <li class="list__item">
+            <div class="service__card">
+              <div class="service__card-header">
+                <img
+                  src="./assets/notebook.svg"
+                  class="service__card-img"
+                  alt=""
```

Comment:
It's always good practice to describe your image content by using an alt attribute just in case the server could not be able to load the image